optuna_dashboard/ts/components/PreferentialTrials.tsx

```diff
@@ -102,19 +177,34 @@ export const PreferentialTrials: FC<{ studyDetail: StudyDetail | null }> = ({
       }
     })
   }
+  const shuffleFlag = shuffleTrial + 20 < displayTrials.last_number
+  if (shuffleFlag) {
+    setShuffleTrial(displayTrials.last_number)
+  }
 
   return (
-    <Box sx={{ display: "flex", flexDirection: "row", flexWrap: "wrap" }}>
-      {displayTrials.numbers.map((t, index) => (
-        <PreferentialTrial
-          key={index}
-          trial={studyDetail.best_trials.find((trial) => trial.number === t)}
-          studyDetail={studyDetail}
-          hideTrial={() => {
-            hideTrial(t)
-          }}
-        />
-      ))}
+    <Box>
```

Comment:
I think you should put some paddings inside the content.

<img width="1579" alt="Screenshot 2023-08-25 17 12 17" src="https://github.com/optuna/optuna-dashboard/assets/5564044/9bef0858-33cf-4fc0-81e7-26f7a6c51fef">